tests/Squidex.Domain.Apps.Core.Tests/Model/Contents/DefaultContentWorkflowTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿// ==========================================================================
+//  Squidex Headless CMS
+// ==========================================================================
+//  Copyright (c) Squidex UG (haftungsbeschränkt)
+//  All rights reserved. Licensed under the MIT license.
+// ==========================================================================
+
+using System.Threading.Tasks;
+using Squidex.Domain.Apps.Core.Contents;
+using Squidex.Domain.Apps.Entities.Contents;
+using Squidex.Domain.Apps.Entities.Schemas.State;
+using Xunit;
+
+namespace Squidex.Domain.Apps.Core.Model.Contents
+{
+    public class DefaultContentWorkflowTests
+    {
+        private static readonly DefaultContentWorkflow Workflow = new DefaultContentWorkflow();
+
+        [Fact]
+        public void Get_initial_status_async_tests()
```

Comment:
Please use the "Should_ ... " naming for tests

`Should_draft_as_initial_state` or so